Jenkinsfile

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env groovy
```

Comment:
why would you need that?

Reply:
I _think_ some people use this to have the file recognized by IDEs and such as groovy, though I don't personally use it.

Reply:
just copied from other plugins. This help syntax highlight on IDEs

Reply:
I use it so I get some highlight across editors (Atom and IJ)

Reply:
Yes, but I personnally do not love this since it kind of conveys Pipeline is Groovy, when IMO we should tell users the very contrary. But not feeling that strongly though (or I would have blocked this).